PhotDM.tex

```diff
@@ -487,21 +493,17 @@ \section{Photometry Data Model} \label{datamodel}
 \par
 
 %%%%%%%%%%%%%%%%%%%% Figure/Image No: 22 starts here %%%%%%%%%%%%%%%%%%%%
-% -- MireilleLouys
-% I would prefer to use the diagram I provided in Modelio,
-% because it traces more details on the associations, and contains the 
-% descriptions inserted in
-% the generated VO-DML xml file
-% the file for replacement will be 
-% "./schema/PhotDMv1-1-overview diagram-2021-04-12.png"
-% -- Mireille Louys
 \begin{figure}[H]
-%\includegraphics[angle=90,width=5.98in,height=7.19in]{./schema/PhotDMv1-1.png}
-% update Mireille new diagram
-\includegraphics[angle=90,width=5.98in,height=7.19in]{./schema/PhotometryOverviewDiagram_jan22.png}
+% update Mireille new diagram Proposed REC MCD comments 
+\includegraphics[angle=90,width=5.98in,height=7.19in]{./schema/PhotometryOverviewDiagram_20220520.png} 
 \caption{Overview class diagram of the Photometry Data Model maintained with Modelio 3.8.}
 \end{figure}
 
+\begin{figure}[H]
+% update Mireille new diagram Proposed REC MCD comments : explain base types and derived types UCD and ISOTime
+\includegraphics[angle=0,width=5.98in ]{./schema/BaseDataTypesDiagram_PR_20220520.png} 
+\caption{Attributes in the PHOTDM classes belong to the base data types defined in the ivoa: VODML template model which is extended here by two classes : UCD for the UCD semantic tag, and ISOTime for time stamps defined and formatted following the DALI specification \citep{2017ivoa.spec.0517D} .} 
```

Comment:
Please do not add files with version-dependent tags like "PR" to git repos -- if accepted, they'd be in there forever, and then the "PR" becomes really confusing.  And in general, don't have dates either: The version control metadata is a much more reliable info on this kind of thing.  That particular file is missing, by the way, so I can't build the document at this point.

Also, please don't have lines longer than 80 characters.  It'll only cause unnecessary conflicts and make diffs a lot harder to read.